sonic-utilities-data/bash_completion.d/spm

```diff
@@ -0,0 +1,8 @@
+_sonic_package_manager_completion() {
+    COMPREPLY=( $( env COMP_WORDS="${COMP_WORDS[*]}" \
```

Comment:
Why not soft link or copy file during install ?

Reply:
Done